modules/Staff/src/Messages/VideoChatRequest.php

```diff
@@ -1,7 +1,7 @@
 <?php
 /*
 Gibbon, Flexible & Open School System
-Copyright (C) 2010, Ross Parker
+Copyright (C) 2022, Cui wenlan
```

Comment:
Please maintain the original copy right owner.

Reply:
OK.
I am also implementing Rocketchat integration now.
Should I insert the original copyright owner for new added files?

Reply:
1. Please keep separated features in separated pull request. It's better for reviewers that way.
2. Please insert the original copyright owner on new files.

Reply:
OK, I will follow you.
I've created v25.0.01 branch.
Let me delete current pull request and recreate it.